docs/integrations.md

```diff
@@ -0,0 +1,17 @@
+---
+title: Integrations
+type: docs
+menu: thanos
+slug: /integrations.md
+---
+
+# Integrations 
+
```

Comment:
Let's add `## StoreAPI` headline. There might be more types of integrations (:

Reply:
Also we have broken links as found by CI: https://circleci.com/gh/improbable-eng/thanos/3979?utm_campaign=vcs-integration-link&utm_medium=referral&utm_source=github-build-link